website/src/hooks/useAlgolia.js

```diff
@@ -14,28 +13,33 @@ const useAlgolia = (inputRef, options = {}) => {
   }, []);
 
   if (!process.env.ALGOLIA_API_KEY || !process.env.ALGOLIA_INDEX) {
```

Comment:
a question: does this env get bundled at build time? do we need to configure the build process to have this set up somehow?

Reply:
Yes. We need to configure this at build time. Does a webpack replace text basically.